src/core/probe/kernel/kernel.rs

```diff
@@ -59,6 +59,12 @@ pub(crate) struct KernelEvent {
     pub(crate) stack_trace: Option<Vec<String>>,
 }
 
+impl EventFmt for KernelEvent {
+    fn event_fmt(&self, f: &mut fmt::Formatter, _: DisplayFormat) -> fmt::Result {
+        write!(f, " {:?}", self)
```

Comment:
don't we need to remove the space now that Event::event_fmt is in charge of high level formatting?

Reply:
Saw that it's removed in a later commit. Ignore this.